ansible/roles/k3s/tasks/runtime.yml

```diff
@@ -0,0 +1,64 @@
+---
+- name: Check if k3s agents are already connected
+  service_facts:
+  register: services_state
```

Comment:
Can we
- prefix stuff with role name to avoid clashes
- given its only intended to be used internally I'd prefix with `_
- un-confuse it from the very similarly named services_states`

So maybe, `_k3s_services_facts`? I try to include a hint re. the task type (or, the command when using shell/command modules) when registering outputs.